### PR TITLE
Initialize Kurdish section and remove broken link

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Review by kt3k — APPROVED** [review]
Nice! LGTM

File: README.md [hunk 1]
```diff
@@ -405,3 +405,8 @@ Consider submitting to the [deno.land/x](https://deno.land/x/) registry.
 ### Darija
 
 - [A first look at Deno | BlaBlaConf 2021 🇲🇦](https://www.youtube.com/watch?v=Y_etUvzAa4s) 
+
+### Kurdish (Central)
+
+- [deno.land](https://deno-ckb.vercel.app)
```
Comment (kt3k) [inline README.md:411]: The link url and link text might better to be `https://deno.roj.im/`?

Comment (roj1512) [reply README.md:411]: If it changes in the future (for example if the translations transferred to another person), the vercel link will still work.

Comment (kt3k) [reply README.md:411]: Ok. let's keep it as is

**Comment by kt3k** [comment]
Hey @roj1512 https://deno-ckb.vercel.app/ seems unavailable now (but https://deno.roj.im/ works). Did you migrate the website to somewhere else?

**Comment by roj1512** [comment]
Hey, I didn't touch anything. It should be a Deno Deploy problem.

image

**Comment by roj1512** [comment]
Yes, I migrated it to Deno Deploy. See #2026.

**Comment by kt3k** [comment]
Ah, Ok. I'll update the url

**Comment by roj1512** [comment]
Please update to denoland-ckb.deno.dev.

**Comment by kt3k** [comment]
Thanks for the url! 👍

**Pull request merged**